campaigns/mobile/mobile-snippet-widget-200.html

```diff
@@ -327,6 +327,11 @@ <h2 class="thank-you hidden" id="thank-you-message-{{ snippet_id }}">{{ thank_yo
          var button = document.querySelector('#firefox-mobile-activation-{{ snippet_id }} .submit-button');
          var input_type;
 
+         // Bug 1238092. Can be removed when Bug 1238092 gets fixed.
+         input.addEventListener('keypress', function(event) {
+             event.stopPropagation();
```

Comment:
Exactly as you say: captures the keypress and yes we'll have to keep this around for a few months